### ISSUE TITLE
Devel-Cover-1.18: test failures with perl 5.22.0

### ISSUE THREAD

**Issue by eserte** [pjcj/Devel--Cover#126]
See http://matrix.cpantesters.org/?dist=Devel-Cover%201.18 for a fail/pass report overview.

**Comment by jkeenan** [comment]
I have also encountered this failure to build with perl 5.22 and have sent pjcj a gzip of the 'cpanm' build log.

**Comment by pjcj** [comment]
This should now be fixed with the release of 1.19.

Thanks for the reports!

**Comment by jkeenan** [comment]
```bash
$ perlbrew use perl-5.22.0
$ cpanm Devel::Cover
Constants from lexical variables potentially modified elsewhere are deprecated at /loader/0x166ef48/aliased.pm line 1.
--> Working on Devel::Cover
Fetching http://www.cpan.org/authors/id/P/PJ/PJCJ/Devel-Cover-1.19.tar.gz ... OK
Configuring Devel-Cover-1.19 ... OK
Building and testing Devel-Cover-1.19 ... OK
Successfully installed Devel-Cover-1.19
1 distribution installed
```
Thank you very much for your attention to this.  I have several releases I'm working on and it will be good to test them and get coverage analysis on perl-5.22.0.
